cluster/examples/composition/azure-sqlserver/definitions/networkgroup.yaml

```diff
@@ -0,0 +1,155 @@
+---
+# This allows Crossplane to manage our new "NetworkGroup" composite resource and
+# the resources it composes.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Looks like we will need to build in RBAC generation into the composition controllers 👍

Reply:
I thought about that, but I don't know whether it would be good practice for Crossplane to automatically grant itself access to things. Would Kubernetes even allow that?